conf/application.conf

```diff
@@ -95,7 +102,7 @@ nomis.payload.token=${?NOMIS_PAYLOAD_TOKEN}
 nomis.private.key="NomisPrivateKey"
 nomis.private.key=${?NOMIS_PRIVATE_KEY}
 
-ldap.string.format="cn=%s,cn=Users,dc=moj,dc=com"
+ldap.string.format="cn=%s,ou=Users,dc=moj,dc=com"
 ldap.string.format=${?LDAP_STRING_FORMAT}
```

Comment:
switched to the live version of these templated for ease of running locally against a test environment